### PR TITLE
added getState prop for react-redux-toastr 7.6

### PR DESCRIPTION
**Pull request by jfbloom22**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/diegoddox/react-redux-toastr/releases/tag/v7.6.0
- [ x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by jfbloom22** [comment]
@kulmajaba @artyomsv Please review when you get a chance.

File: types/react-redux-toastr/index.d.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ interface BasicToastrOptions {
     timeOut?: number;
     transitionIn?: transitionInType;
     transitionOut?: transitionOutType;
+    getState?:(state: ToastrState)=>ToastrState;
```
Comment (jessetrinity) [inline types/react-redux-toastr/index.d.ts]: The whitespace here doesn't look like it matches the rest of the file.

**Review by jessetrinity — CHANGES_REQUESTED** [review]

File: types/react-redux-toastr/index.d.ts (hunk 1 above)
Comment (jfbloom22) [reply types/react-redux-toastr/index.d.ts]: Thanks.  I have gotten too used to my Prettier plugin fixing things like this on commit.  I updated it.  Please review.

**Review by jessetrinity — APPROVED** [review]

**Comment by jessetrinity** [comment]
I don't know why it decided to start failing :(

**Comment by jfbloom22** [comment]
yeah no idea why the build failed...

**Comment by jessetrinity** [comment]
Looks like we bumped a version too early.

**Comment by jfbloom22** [comment]
looks like it is failing here:
```
Installing to /home/vsts/.dts/typescript-installs/3.7...
npm ERR! code ETARGET
npm ERR! notarget No matching version found for typescript@3.7.
```
is this something I can fix in my pull request?

**Comment by jessetrinity** [comment]
@jfbloom22 nope, you're all good. Completely on our end.

**Pull request merged**